configuration/transitions.md

```diff
@@ -427,3 +428,24 @@ Supported `events_types` are:
     ]
   }
 ```
+
+### update_patient
+
+Updates a `data_record` to set its patient to its sender, the resulting doc will have `fields.patient_uuid` and `fields.patient_id` filled with the sender's information. Provides hydrated patient information to subsequent transitions. 
```

Comment:
```suggestion
Updates a `data_record` to set its patient to its sender. The resulting doc will have `fields.patient_uuid` and `fields.patient_id` filled with the sender's information. Provides hydrated patient information to subsequent transitions. 
```
You somewhat explain it the next paragraph, but consider explicitly stating that "sender" is the person who is attached to the phone number the original SMS was sent from.